spec/support/mock_api.rb

```diff
@@ -0,0 +1,48 @@
+module AnsibleTowerClient
```

Comment:
I thought about this for a couple days and feel like it's better to keep all components of the gem inside the gem namespace.